src/mixins/data-iterable.js

```diff
@@ -240,11 +240,18 @@ export default {
         this.defaultPagination = updatedPagination
       }
     },
+    getItemKey (item) {
+      const key = this.itemKey
+      if (typeof key === 'function') {
+        return key(item)
+      }
+      return getObjectValueByPath(item, key)
```

Comment:
change to:
```
const key = this.itemKey

return typeof key === 'function'
  ? key(item)
  : getObjectValuesByPath(item, key)
```

Reply:
This could be also a computed prop
`return typeof key === function ? key : item => getObjectValuesByPath(Item, key)`